src/test/java/core/basesyntax/storage/StrategySubtractionTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.storage;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.servise.FruitRecordDto;
+import core.basesyntax.servise.Operation;
+import core.basesyntax.servise.exception.InvalidValueOfAmountException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StrategySubtractionTest {
+    private static final Fruit BANANA = new Fruit("banana");
```

Comment:
```suggestion
    private static final Fruit banana = new Fruit("banana");
```
it's not considered to be constant because class is not deeply immutable

Reply:
done